src/sample.ts

```diff
@@ -0,0 +1,4 @@
+// tslint:disable-next-line:variable-name
+const MiniApp = require('./miniapp');
+
+console.log(`Unique Id: ${MiniApp.getUniqueId()}`);
```

Comment:
What is this sample for? Just for demonstrating how to use the SDK? I think that once we have the sample Mini App it will act as the sample of how to use the. SDK, so maybe we don't need this file. What do you think?

Reply:
Agree. Let's me remove those relevant sample files.